v2/pkg/operators/common/dsl/dsl.go

```diff
@@ -339,6 +370,24 @@ func init() {
 				return strings.Join(stringElements, separator), nil
 			},
 		),
+		"join_slice": makeDslWithOptionalArgsFunction(
+			"(separator string, elements ...interface{}) string",
+			func(arguments ...interface{}) (interface{}, error) {
+				argumentsSize := len(arguments)
+				if argumentsSize != 2 {
+					return nil, errors.New("incorrect number of arguments received")
+				}
+
+				separator := types.ToString(arguments[0])
+				elements := arguments[1].([]string)
+
+				stringElements := make([]string, 0, len(elements))
+				for _, element := range elements {
+					stringElements = append(stringElements, types.ToString(element))
+				}
+				return strings.Join(stringElements, separator), nil
```

Comment:
`elements` is already `[]string`, you can use it directly instead of `stringElements`

Reply:
Done.